src/main/java/io/strimzi/kafka/bridge/http/converter/JsonUtils.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.http.converter;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.vertx.core.buffer.Buffer;
+
+import java.io.IOException;
+
+/**
+ * Provides some utility methods for JSON encoding/decoding
+ */
+public class JsonUtils {
+
+    /**
+     * ObjectMapper instance used for JSON encoding/decoding
+     */
+    public static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Does this need to be public? It seems to be used only here.

Reply:
Good catch, I will fix it.